plenum/server/replica.py

```diff
@@ -2436,3 +2443,15 @@ def _remove_ordered_from_queue(self, last_caught_up_3PC=None):
             removed.insert(0, self.outBox[i])
             del self.outBox[i]
         return removed
+
+    def _get_last_timestamp_from_state(self, ledger_id):
+        last_timestamp = None
+        if ledger_id == DOMAIN_LEDGER_ID:
```

Comment:
I don't like too deep if-else hierarchy (more than 2). 
I think the following pattern is more readable:
```
if not condition1:
    return
do_something()
if not condition2:
    return
....
```
Not so critical for this particular method since it's not so big.